dotCMS/src/main/java/com/dotcms/publishing/XMLSerializerUtil.java

```diff
@@ -1,20 +1,21 @@
 package com.dotcms.publishing;
 
 
+import com.dotcms.publisher.business.PublishAuditHistory;
+import com.dotcms.publisher.pusher.PushPublisherConfig;
+import com.dotcms.publisher.util.dependencies.PushPublishigDependencyProcesor;
 import com.thoughtworks.xstream.XStream;
 import com.thoughtworks.xstream.XStreamException;
 import com.thoughtworks.xstream.io.HierarchicalStreamWriter;
 import com.thoughtworks.xstream.io.xml.DomDriver;
-import java.io.OutputStream;
-import java.io.OutputStreamWriter;
 
 /**
  * Util for serialize object to XML file
  */
 public class XMLSerializerUtil {
 
     private static final XMLSerializerUtil instance= new XMLSerializerUtil();
-    private XStream xmlSerializer = null;
```

Comment:
It seems you still using XStream

Reply:
@jdotcms I need to. For publishing purposes it serializes to a XML representation of several objects. Mainly `PublishAuditHistory` and `PushPublisherConfig`. I switched to JSON just for the manifest.